lib/ruby_snowflake_client.rb

```diff
@@ -1,23 +1,27 @@
 # frozen_string_literal: true
 
-require "ruby_snowflake_client_ext" # build bundle of the go files
+#require "ruby_snowflake_client_ext" # build bundle of the go files
+require_relative "../ext/ruby_snowflake_client_ext" # build bundle of the go files
 
 module Snowflake
+  class Client
+    def connect(account:"", warehouse:"", database:"", schema: "", user: "", password: "", role: "")
```

Comment:
Good place to leave a comment w/ that info you put in the PR?
```suggestion
    # wrap a private _connect method, as sending kwargs to Go would require wrapping the function in
    # C as the current CGO has a limitation on accepting variadic arguments for functions.
    def connect(account:"", warehouse:"", database:"", schema: "", user: "", password: "", role: "")
```